tests/infra/remote.py

```diff
@@ -418,45 +435,60 @@ def __init__(
             else "{}.ledger".format(node_id)
         )
 
-        cmd = [
-            self.BIN,
-            "--enclave-file={}".format(lib_path),
-            "--raft-election-timeout-ms={}".format(election_timeout),
-            "--raft-host={}".format(host),
-            "--raft-port={}".format(raft_port),
-            "--tls-host={}".format(host),
-            "--tls-pubhost={}".format(pubhost),
-            "--tls-port={}".format(tls_port),
-            "--ledger-file={}".format(self.ledger_file_name),
-            "--node-cert-file={}".format(self.pem),
-            "--enclave-type={}".format(enclave_type),
-            "--log-level={}".format(log_level),
-        ]
-
-        if sig_max_tx is not None:
-            cmd += ["--sig-max-tx={}".format(sig_max_tx)]
-
-        if sig_max_ms is not None:
-            cmd += ["--sig-max-ms={}".format(sig_max_ms)]
-
-        if memory_reserve_startup is not None:
-            cmd += ["--memory-reserve-startup={}".format(memory_reserve_startup)]
-
-        if notify_server is not None:
-            notify_server_host, *notify_server_port = notify_server.split(":")
-
-            if not notify_server_host or not (
-                notify_server_port and notify_server_port[0]
-            ):
+        cmd = [self.BIN, "--enclave-file={}".format(lib_path)]
+
+        # If the remote needs to verify the quote, only a subset of arguments are required
+        if self.verify_quote:
+            cmd += ["--start=verify"]
+
+            if other_quote is None or other_quoted_data is None:
```

Comment:
if not other_quote or not other_quoted_data is probably enough here, but it's best to split the check to give a more specific exception about which is missing.